test/examples/ucx_hello_world.h

```diff
@@ -35,6 +48,72 @@
     } while (0)
 
 
+#define CUDA_FUNC(_func)                                   \
```

Comment:
should it be under `#ifdef UCX_HELLO_WORLD_CUDA` as well?
How did it compile on machines without cuda?